src/basic/pidref.h

```diff
@@ -3,7 +3,7 @@
 
 #include "macro.h"
 
-/* An embeddable structure carrying a reference to a process. Supposed to be used when tracking processes continously. */
```

Comment:
this is a very annoying word to spell, can we file a bug on the english bug tracker?